src/app/user-profile/user-info/user-info.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="flex-end button-wrapper">
+  <app-button className="icon-button">
+    <span class="icon-button__plus"> </span>
+  </app-button>
+  <app-button className="icon-button">
+    <span class="icon-button__edit"></span>
+  </app-button>
+</div>
+
+<div class="user-wrapper">
+  <div class="flex-center user">
+    <div class="flex user__info">
+      <div class="user__info-image">
+        <img
+          class="user-image"
+          src="../../../assets/images/user-image.png"
+          alt="user name"
+        />
+      </div>
+
+      <div class="user__info-name">
+        <h2 class="user-name">
+          {{ loggedInUser?.firstName + " " + loggedInUser?.lastName }}
+        </h2>
+        <div class="user-occupation">Collaborator & Editor</div>
+      </div>
+    </div>
+
+    <div class="user__description">
+      Meet Jonathan Doe, a passionate writer and blogger with a love for
+      technology and travel. Jonathan holds a degree in Computer Science and has
+      spent years working in the tech industry, gaining a deep understanding of
+      the impact technology has on our lives.
```

Comment:
Keep the text in a variable in the ts file and render it here using interpolation